README.md

```diff
@@ -15,18 +15,16 @@ broker/server listening on port 5672:
 
 ```js
 var container = require('rhea');
-container.on('connection_open', function (context) {
```

Comment:
Could also change this to container.once('connection_open'...), however your change is also fine.